graph/is_graph_bipartite.cpp

```diff
@@ -0,0 +1,161 @@
+/**
+ *  @file is_graph_bipartite
```

Comment:
```suggestion
 *  @file 
```
No need to add a file name. If adding it, the name **must** match, including the suffix. To avoid any issues, better not to specify filename.